README.md

```diff
@@ -252,7 +252,7 @@ Returns a cursor instead of an array as `db.collection.aggregate` does.
 
 See https://docs.mongodb.org/manual/reference/method/db.collection.aggregate/
 
-#### `db.collection.count([query])`
+#### `db.collection.count([query], [options])`
```

Comment:
Maybe note that a subset of [these options](https://www.mongodb.com/docs/manual/reference/method/db.collection.count/#:~:text=The%20options%20document) are supported?

Reply:
Good one! Added [here](https://github.com/mongoist/mongoist/pull/96/commits/5317274b5b4e4fb76a074d1c53381496b2c8ae65)